src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -281,21 +288,36 @@ class GlobalSelectionHeader extends React.Component {
       .slice(0, 1);
   };
 
+  getBackButton = () => {
+    return (
+      <BackToIssues
+        to={`/organizations/${this.props.organization.slug}/issues/${window.location
```

Comment:
There may be one edgecase here... if you're in `Events` and you're searching for events, you may not want to persist `query`... are we just trying to preserve the header properties here?